indico/modules/events/contributions/models/contributions.py

```diff
@@ -65,6 +65,8 @@ class Contribution(DescriptionMixin, ProtectionManagersMixin, LocationMixin, Att
     __auto_table_args = (db.Index(None, 'friendly_id', 'event_id', unique=True),
                          db.Index(None, 'event_id', 'track_id'),
                          db.Index(None, 'event_id', 'abstract_id'),
+                         db.Index('ix_uq_abstract_id', 'abstract_id', unique=True,
+                                  postgresql_where=db.text('NOT is_deleted')),
```

Comment:
might fit in one line; and i believe that index name will be autogenerated so you can just pass `None` for the name